src/styles/scss/main.scss

```diff
@@ -0,0 +1,121 @@
+body {
+  background-color: #f5f6fa;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.App {
+  margin: 100px;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header {
+  display: flex;
+  padding: 0 100px;
+  font-size: 15px;
+  color: #0290ff;
+  font-weight: bold;
+  align-items: center;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+
+  ul {
+    display: flex;
+
+    li {
+      margin: 0 20px;
+      list-style: none;
+    }
+  }
+
+  #books {
+    font-size: 13px;
+    color: #121212;
+    letter-spacing: 1.9px;
+  }
+
+  #categories {
+    opacity: 0.5;
+    font-size: 13px;
+    color: #121212;
+    letter-spacing: 1.9px;
+  }
+}
+
+.book-div {
+  display: flex;
+  height: 170px;
+  border-radius: 4px;
+  margin: 20px 100px;
+  align-items: center;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  justify-content: space-between;
+}
+
+.book-div p {
+  margin: 16px 27px;
+  width: 100%;
+}
+
+.book-div button,
+.book-form button {
+  font-size: 25px;
+  color: #fafafa;
+  margin: 16px 20px;
+  font-weight: bold;
+  text-transform: uppercase;
+  background-color: #0290ff;
```

Comment:
you might want to apply the following;
1. `border-style: none`
2. `cursor: pointer`

before::
![image](https://user-images.githubusercontent.com/31897434/83095227-b167d080-a09a-11ea-8381-b55a846fd91f.png)


after:: 
![image](https://user-images.githubusercontent.com/31897434/83095105-8a110380-a09a-11ea-87e8-b809f7f92a85.png)